Net/src/HTTPServerResponseImpl.cpp

```diff
@@ -128,7 +128,20 @@ void HTTPServerResponseImpl::sendFile(const std::string& path, const std::string
 		write(*_pStream);
 		if (_pRequest && _pRequest->getMethod() != HTTPRequest::HTTP_HEAD)
 		{
+#ifdef POCO_USE_SENDFILE_FOR_HTTPSERVER
+			delete _pStream; // delete the stream to flush the HTTP headers to the socket, required by HTTP 1.0 and above
+			_pStream = nullptr;
+
+			Poco::Int64 sent = 0;
+			Poco::Int64 offset = 0;
+			while (sent < length)
```

Comment:
This can stay here for ever because _sendfile() is so buggie. It does not do prober error handling at Linux at least. We definitely need to look closer how to fix that before start using that. I also do not understand why it returns Int64. We example use 32 bit systems so it does not make much sense to force 64 bit return value.

Reply:
If we have any doubts about that we'll leave it out.

Reply:
@teksturi , thank you for review . I'll fix logical mistake with ```sent``` and ```offset```

Reply:
And about Int64...
Poco hasn't platform dependent integral types like size_t and ssize_t and I made a decision that Int64 is a compromise

Reply:
I hope that now code more correct. Please review it